### PR TITLE
[ServiceNow v2] Fixed issue in mirroring

### PR DESCRIPTION
**Pull request by mmhw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-33707)

## Description
Fixed an issue where the **get-remote-data** (mirroring) command was not returning the display values for all fields when the *Use display values* option was enabled.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py [hunk 1]
```diff
@@ -547,23 +547,20 @@ def split_notes(raw_notes, note_type, time_info):
     return notes
 
 
-def convert_to_notes_result(full_response, time_info) -> dict:
+def convert_to_notes_result(ticket, time_info) -> dict:
     """
     Converts the response of a ticket to the response format when making a query for notes only.
```
Comment (bziser) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:552]: need to be changed?
please add DocString and types to the args

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py [hunk 2]
```diff
@@ -2435,18 +2465,18 @@ def parse_dict_ticket_fields(client: Client, ticket: dict) -> dict:
     return ticket
 
 
-def get_timezone_offset(full_response, display_date_format):
+def get_timezone_offset(ticket, display_date_format):
```
Comment (bziser) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py:2468]: please add types

**Review by bziser — COMMENTED** [review]
looks good

**Comment by bziser** [comment]
@mmhw 
please add a test which simulates the bug

**Review by bziser — APPROVED** [review]
looks good
It would be helpful if you could provide screenshots before and after the changes were made to the mapper
And update the description of the pr

**Comment by mmhw** [comment]
**Mapper changes**
Before:
image

After:
image

**Pull request merged**